sources/accessControlLib.move

```diff
@@ -271,9 +286,25 @@ module peeranha::accessControlLib {
         revokeRole_(userRolesCollection, communityModeratorRole, userId);
     }
 
+    //
+    fun isBannedUser(userRolesCollection: &UserRolesCollection, userId: ID, communityId: ID) {
```

Comment:
Functions that have name starting with 'is' are expected to return boolean. In  this case the function does not return but checks/verifies.

I suggest to rename it to `checkUserNotBanned`.